conjure-undertow-processor/src/test/java/com/palantir/conjure/java/undertow/processor/sample/DefaultDecoderService.java

```diff
@@ -32,49 +31,55 @@
 import java.util.Set;
 import java.util.UUID;
 
+@SuppressWarnings("TooManyArguments")
 public interface DefaultDecoderService {
 
     @Handle(method = HttpMethod.GET, path = "/queryParam")
     String queryParam(
-            @Handle.QueryParam(value = "stringParam") String stringParam,
-            @Handle.QueryParam(value = "booleanParam") Boolean booleanParam,
-            @Handle.QueryParam(value = "stringSetParam") Set<String> stringSetParam,
-            @Handle.QueryParam(value = "stringListParam") List<String> stringListParam,
-            @Handle.QueryParam(value = "optionalStringParam") Optional<String> optionalStringParam,
-            @Handle.QueryParam(value = "decoderParam", decoder = StringCollectionDecoder.class) String decoderParam);
-
-    @Handle(method = HttpMethod.GET, path = "/moreQueryParams")
-    String moreQueryParams(
```

Comment:
Suppressed the `TooManyArguments` so we don't have to awkwardly split the query params endpoint.